packages/lib/import-enex.ts

```
@@ -471,19 +476,25 @@ const parseNotes = async (parentFolderId: string, filePath: string, importOption
 
-					const body: string = importOptions.outputFormat === 'html' ?
-						await enexXmlToHtml(note.bodyXml, note.resources) :
-						await enexXmlToMd(note.bodyXml, note.resources, note.tasks);
+					let outputFormat = importOptions.outputFormat;
+
+					if (outputFormat === 'md' && note.bodyXml && isHtmlNote(note.bodyXml)) {
+						outputFormat = 'html';
+					}
+
+					const body: string = outputFormat === 'html'
+						  ? await enexXmlToHtml(note.bodyXml, note.resources)
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)